sdk/storage/Azure.Storage.Queues/src/Models/QueueMessage.cs

```diff
@@ -14,6 +14,11 @@ public partial class QueueMessage
     {
         internal QueueMessage() { }
 
+        internal QueueMessage(string messageText)
+        {
+            MessageText = messageText;
+        }
```

Comment:
where is this constructor used?

Reply:
Never.  The new generator automatically makes a bunch of public constructors we don't want, this is how you tell the generator not to do that.

Reply:
Sounds like feature request for generator. @pakrym

Reply:
What's the feature request?

Reply:
It seems that we've added bunch of code to hide constructors. It would be great to tell generator to either not generate them or hide them (or maybe we need some readme transform that's already available).

Reply:
Interesting, this particular ctor shouldn't have been public in the first place. Are there any operations that take  `QueueMessage` as an input?